external/anomaly/anomaly_classification/configs/padim/configuration.yaml

```diff
@@ -1,40 +1,29 @@
-description: Configuration for an anomaly classification task
-header: Configuration for an anomaly classification task
-type: CONFIGURABLE_PARAMETERS
-visible_in_ui: true
-
-model:
-  description: Model Parameters
-  header: Model Parameters
-  type: PARAMETER_GROUP
-  visible_in_ui: true
-  name:
-    affects_outcome_of: TRAINING
-    default_value: padim
-    description: The name of the anomaly classification algorithm.
-    editable: True
-    header: Model Name
-    enum_name: AnomalyModel
-    options:
-      PADIM: padim
-    type: SELECTABLE
+dataset:
+  description: Dataset Parameters
+  header: Dataset Parameters
+  num_workers:
+    affects_outcome_of: NONE
+    default_value: 8
+    description: Increasing this value might improve training speed however it might
+      cause out of memory errors. If the number of workers is set to zero, data loading
+      will happen in the main training thread.
+    editable: true
+    header: Number of workers
+    max_value: 8
+    min_value: 0
+    type: INTEGER
     ui_rules:
       action: DISABLE_EDITING
       operator: AND
       rules: []
       type: UI_RULES
-    value: padim
-    visible_in_ui: True
+    value: 8
+    visible_in_ui: true
     warning: null
-
-dataset:
-  description: Dataset Parameters
-  header: Dataset Parameters
   train_batch_size:
     affects_outcome_of: TRAINING
-    default_value: 32
-    description:
-      The number of training samples seen in each iteration of training.
+    default_value: 5
```

Comment:
Why 5?

Reply:
I just re-used the default values that were already in the previous `configuration.yaml`. I agree that something like 16 or 32 would be better.